lib/spack/spack/test/cache_fetch.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
Is this file supposed to be checked in for this PR?

Reply:
I added it to try to get the coverage numbers up, before I asked if I should be spending time on that effort.

Should I move it to a separate PR?

Reply:
@tgamblin Should I move the new `cache_fetch` tests -- I'd added to improve coverage a bit -- to a separate PR?